composeApp/src/iosMain/kotlin/org/ooni/testing/factories/DatabaseHelper.kt

```diff
@@ -0,0 +1,349 @@
+package org.ooni.testing.factories
+
+import org.ooni.engine.models.NetworkType
+import org.ooni.engine.models.TaskOrigin
+import org.ooni.engine.models.TestType
+import org.ooni.probe.config.OrganizationConfig
+import org.ooni.probe.data.models.InstalledTestDescriptorModel
+import org.ooni.probe.data.models.MeasurementModel
+import org.ooni.probe.data.models.NetworkModel
+import org.ooni.probe.di.Dependencies
+import kotlin.concurrent.Volatile
+
+class DatabaseHelper private constructor(private val dependency: Dependencies) {
```

Comment:
Could this file be shared with the Android screenshots, inside `commonTest`? To avoid code duplication.

Reply:
I haven't found a way to access the code in `commonTest` from `iOS` source path.

Reply:
This is what I had done on Android instrumented tests to do it. 

<img width="758" alt="Screenshot 2025-03-31 at 12 28 27" src="https://github.com/user-attachments/assets/9f1f80a3-4821-4eec-933d-b14e3b41e906" />

But the symlinks worked for the factories no?

Reply:
the symlinks worked.